### PR TITLE
fix: remove duplicated logs introduced by adding logger in manager

### PR DESCRIPTION
**Pull request by blackpiglet**
Remove duplicated log entries, which are introduced by logger added in controller manager in pull request #4341.

Fix issue #4368 

Signed-off-by: Xun Jiang <jxun@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/controller/backup_storage_location_controller.go [hunk 1]
```diff
@@ -59,8 +60,8 @@ func (r *BackupStorageLocationReconciler) Reconcile(ctx context.Context, req ctr
 
 	locationList, err := storage.ListBackupStorageLocations(r.Ctx, r.Client, req.Namespace)
 	if err != nil {
-		log.WithError(err).Error("No backup storage locations found, at least one is required")
-		return ctrl.Result{}, err
+		return ctrl.Result{},
+			fmt.Errorf("error: %s, no backup storage locations found, at least one is required", err.Error())
```
Comment (dsu-igeek) [inline pkg/controller/backup_storage_location_controller.go:64]: Let's make sure the stack trace is returned with the error here.

**Review by dsu-igeek — CHANGES_REQUESTED** [review]
Please make sure the error stack gets returned with the error

File: pkg/controller/backup_storage_location_controller.go (hunk 1 above)
Comment (blackpiglet) [reply pkg/controller/backup_storage_location_controller.go]: Got it. Updated according to comments.

File: pkg/controller/download_request_controller.go [hunk 2]
```diff
@@ -64,22 +65,23 @@ func (r *DownloadRequestReconciler) Reconcile(ctx context.Context, req ctrl.Requ
...
-		log.WithError(err).Error("Error getting a patch helper to update this resource")
-		return ctrl.Result{}, errors.WithStack(err)
+		return ctrl.Result{},
+			fmt.Errorf("error: %s, error getting a patch helper to update this resource", errors.WithStack(err))
 	}
 
 	defer func() {
 		// Always attempt to Patch the downloadRequest object and status after each reconciliation.
 		if err := patchHelper.Patch(ctx, downloadRequest); err != nil {
 			log.WithError(err).Error("Error updating download request")
```
[the remark this answers is not in the document]
Comment (blackpiglet) [reply pkg/controller/download_request_controller.go:82]: Thanks. I forgot to delete the original log.

File: pkg/controller/download_request_controller.go [hunk 3]
```diff
@@ -91,7 +93,8 @@ func (r *DownloadRequestReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			log.Debug("DownloadRequest has expired - deleting")
 			if err := r.Client.Delete(ctx, downloadRequest); err != nil {
 				log.WithError(err).Error("Error deleting an expired download request")
```
[the remark this answers is not in the document]
Comment (blackpiglet) [reply pkg/controller/download_request_controller.go:95]: Thanks. I forgot to delete the original log.

**Pull request closed without merging**